kitsune/settings.py

```diff
@@ -928,6 +928,7 @@ def JINJA_CONFIG():
     # TODO: Make sure that db number is respected
     "default": config("REDIS_DEFAULT_URL"),
     "helpfulvotes": config("REDIS_HELPFULVOTES_URL"),
+    "account_events": config("REDIS_ACCOUNT_EVENTS_URL"),
```

Comment:
FYI. I did check and any new Redis DB will be automatically created upon initial connection. So there's nothing to worry about there.